### PR TITLE
More tests legacy market

### PR DESCRIPTION
**Pull request by dbeal-eth**
also one change to ensure wsteth address is correct

### CODE REVIEW

File: omnibus-mainnet.toml [hunk 1]
```diff
@@ -34,6 +34,9 @@ defaultValue = "synthetix-legacy-market:3.3.5"
 [setting.susde_package]
 defaultValue = 'susde-token'
 
+[setting.wsteth_package]
+defaultValue = 'wsteth-token'
+
 [setting.owner]
 defaultValue = "0x6E1613B5c68B4Cf2A58400D8019a97849A678139"
```
Comment (kaleb-keny) [inline omnibus-mainnet.toml:41]: this is outdated?

**Comment by kaleb-keny** [comment]
this is not wsteth, can we keep a comment to switch this once chainlink sends us a wsteth feed on mainnet (expected early august)
https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L53

**Comment by kaleb-keny** [comment]
Also worthwhile adding a comment here, that this needs to be updated, once chainlink sends us a susde feed that is redemption based calculation https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L56

**Comment by kaleb-keny** [comment]
need to add the same staleness configs for the rest of the oracles, not only snx https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L65C17-L65C23

**Comment by kaleb-keny** [comment]
let's bump this to 50 snx, 600k gas to liquidate (roughly), at 30 gwei gas price amount to 30 bucks. @ 2$ snx price, 10 snx will not cut it

**Comment by kaleb-keny** [comment]
https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L71C17-L71C36 500%, right?

**Comment by kaleb-keny** [comment]
https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L77C17-L77C59 outdated

**Comment by kaleb-keny** [comment]
are these ccip up to date ?
 https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L92C17-L92C59
 https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L89

**Comment by kaleb-keny** [comment]
https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L101 zero for now

**Comment by kaleb-keny** [comment]
can be deleted https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L104 ?

**Comment by kaleb-keny** [comment]
also outdated https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L107

**Comment by kaleb-keny** [comment]
also outdated https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L110

**Comment by kaleb-keny** [comment]
we can set all the weight on the legacy market for now https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L135

**Review by kaleb-keny — CHANGES_REQUESTED** [review]

**Comment by dbeal-eth** [comment]
> are these ccip up to date ? https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L92C17-L92C59
> 
> https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L89

no effect, we dont use it

**Comment by dbeal-eth** [comment]
> we can set all the weight on the legacy market for now
> 
> https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L135

the spot market will not be able to do anything if it has 0 liquidity, which goes against our plan for having the ability for people to deposit USDe if I remember correctly. Is that fine?

**Comment by dbeal-eth** [comment]
> Also worthwhile adding a comment here, that this needs to be updated, once chainlink sends us a susde feed that is redemption based calculation
> 
> https://github.com/Synthetixio/synthetix-deployments/blob/6112ee3a710390876a195b346e8c1ac032c21bc6/omnibus-mainnet.toml#L56

this is labelled as the sUSDe feed for mainnet on their website. is the feed taht is currently there just missing something important?

**Review by kaleb-keny — APPROVED** [review]

**Pull request merged**